src/event.js

```diff
@@ -610,8 +610,7 @@ jQuery.event = {
 			event,
 			{
 				type: type,
-				isSimulated: true,
-				originalEvent: {}
```

Comment:
> How is jQuery.Event{ type: undefined, timeStamp: <Date>, [jQuery.expando]: true } better than {}, and how is {} better than undefined?

Same way, i guess, it is not better then removing that `originalEvent` property, i'm for the reverting, but only because of the consistency, even if it is pointless, which i'm not totally sure it is.

For example, in standart event model of the DOM you can `preventDefault` or `stopPropagation` focus event, but it is not preventable nor it bubbles, but functions are still there as per spec - http://www.w3.org/TR/DOM-Level-2-Events/events.html#Events-Event-preventDefault

So "If an event is cancelable, the preventDefault method is used to signify that the event is to be canceled...  Calling this method for a non-cancelable event has no effect".

You know it always there, you know it always be consistent no matter what type of event it is, for me this is sounds as a good policy.  

Although this situation is different since user might actually need access to the real event. And as i said - 

>  In any case, focus(in | out) do exist in most browsers and in IE it would exist for real. User might not even know we shim it.